src/renderers/shaders/ShaderChunk/lights_pars.glsl

```diff
@@ -138,8 +140,16 @@
 		float dotNL = dot( geometry.normal, hemiLight.direction );
 		float hemiDiffuseWeight = 0.5 * dotNL + 0.5;
 
+	#if defined( PHYSICALLY_CORRECT_LIGHTS )
+
+		return mix( hemiLight.groundColor, hemiLight.skyColor, hemiDiffuseWeight );
+
+	#else
+
 		return PI * mix( hemiLight.groundColor, hemiLight.skyColor, hemiDiffuseWeight );
```

Comment:
I see a lot of repetition of this code. Can we define this `PI` with a different name? I don't know the specifics, but it would be great if we could just do something like...

```javascript
parameters.physicallyCorrectLights ? "#define LIGHTS_PI 1.0" : '#define LIGHTS_PI PI',
```